sea-query-postgres/src/lib.rs

```diff
@@ -55,12 +96,12 @@ impl ToSql for Value {
             Value::TimeDateTime(v) => v.as_deref().to_sql(ty, out),
             #[cfg(feature = "with-time")]
             Value::TimeDateTimeWithTimeZone(v) => v.as_deref().to_sql(ty, out),
-            #[cfg(feature = "postgres-rust_decimal")]
+            #[cfg(feature = "with-rust_decimal")]
             Value::Decimal(v) => v.as_deref().to_sql(ty, out),
             #[cfg(feature = "with-uuid")]
             Value::Uuid(v) => v.as_deref().to_sql(ty, out),
             #[cfg(feature = "postgres-array")]
-            Value::Array(v) => v.as_deref().to_sql(ty, out),
+            Value::Array(_v) => todo!(),
```

Comment:
Can someone help me with this part?

Reply:
Consider this: originally `ToSql` is implemented by `Value`. And, the `v` of `Value::Array(v)` is `Vec<Value>`. So, it satisfy the blanket implementation of `Vec<T: ToSql>` provided by `postgres`.

However, now, we only implement `ToSql` for `PostgresValue` but not `Value`. That's why we need to clone it as you did on latest commit.